testdata/p4_16_samples_outputs/pna-dpdk-direct-meter-learner.p4.spec

```diff
@@ -33,7 +33,6 @@ struct main_metadata_t {
 	bit<8> local_metadata_timeout
 	bit<32> local_metadata_port_out
 	bit<32> pna_main_output_metadata_output_port
-	bit<32> table_entry_index
```

Comment:
This is a valid test for Direct Meter for tables with add_on_miss. These instructions/metadata field shouldn't be removed

Reply:
fixed